### ISSUE TITLE
JavaCPP does not seems to allow `null` to be passed instead of a ptr.

### ISSUE THREAD

**Issue by StRigaud** [clEsperanto/clesperantoj_prototype#41]
```java
Tier1.absolute(currentDevice, input, output);
```
is fine but
```java
ArrayJ output = Tier1.absolute(currentDevice, input, null);
```
raise an error:
```
Exception in thread "main" java.lang.NullPointerException: Pointer address of argument 2 is NULL.
        at net.clesperanto._internals.kernelj$Tier1.absolute(Native Method)
        at net.clesperanto.kernels.Tier1.absolute(Tier1.java:66)
        at net.clesperanto.Imglib2ClesperantoJExample.main(Imglib2ClesperantoJExample.java:37)
```

**Comment by StRigaud** [comment]
I think this is related to how the function is define in the `native` part:
```cpp
static ArrayJ absolute(const DeviceJ &dev, const ArrayJ &src, ArrayJ &dst);
```
ArrayJ parameters are passed as ref, hence cannot be `null`, we should update the function to take `ptr`.

We would need to rewrite it as followed:
```cpp
static ArrayJ Tier1::absolute( DeviceJ* dev,  ArrayJ* src, ArrayJ* dst)
```
and add a check for `nullptr` inside the cpp code

Doing so seems to work.

**Comment by StRigaud** [comment]
Solved by #42
